common/pylintrc

```diff
@@ -175,6 +175,26 @@ evaluation=10.0 - ((float(5 * error + warning + refactor + convention) / stateme
 comment=no
 
 
+[SPELLING]
+
+# Spelling dictionary name.
+# If this value will be non-empty (e.g. 'en_US') and pyenchant will not be
+# installed, pylint will fail.
+# If this will be left empty pylint will ignore all spelling errors.
+spelling-dict=en_US
+
+# List of comma separated words that should not be checked.
+spelling-ignore-words=
+
+# A path to a file that contains private dictionary; one word per line.
+# Path relative to current working directory.
+spelling-private-dict-file=
```

Comment:
Good point, since they are the same we can rely on that. Let me fix that.

To run pylint manually from other directory it will be enough to provide `--spelling-private-dict-file` with proper path on command line.